src/test/resources/features/CreateRegAndSubE2ETests.feature

```diff
@@ -584,8 +589,11 @@ Feature: User registration and subscription e2e journey
       | KEY          | VALUE          |
       | addressLine1 | Address Line 1 |
       | addressLine3 | City           |
-      | postalCode   | EH5 5WY        |
+      | postalCode   | INVALID       |
       | countryCode  | United Kingdom |
+    Then I should see address error message Enter a full UK postcode on the Postal code Element
```

Comment:
same here:
please move error validation tests to unit tests and remove from ATs